src/sass/_variables.scss

```diff
@@ -1,19 +1,29 @@
-$main-color: #337ab7; // is taken from bootstrap ($brand-primary)
+// colors
+$main-color: #1986ff;
+$default-bg-color: #f1f6f8;
+$rocket-gray: #b7b7b7;
```

Comment:
Actually, I like semantic names better. But @gkatsanos said he prefers the way it is now. Could we talk about pros and cons?